packages/query/src/stacks/balance/account-balance.hooks.ts

```diff
@@ -0,0 +1,60 @@
+import BigNumber from 'bignumber.js';
+
+import type { Money, StxCryptoAssetBalance } from '@leather-wallet/models';
+import { createMoney, subtractMoney, sumMoney } from '@leather-wallet/utils';
+
+import {
+  AccountBalanceStxKeys,
+  AddressBalanceResponse,
+  accountBalanceStxKeys,
+} from '../hiro-api-types';
+import {
+  useMempoolTxsInboundBalance,
+  useMempoolTxsOutboundBalance,
+} from '../mempool/mempool.hooks';
+import { useStacksAccountBalanceQuery } from './account-balance.query';
+
+function createStxMoney(resp: AddressBalanceResponse) {
+  return Object.fromEntries(
+    accountBalanceStxKeys.map(key => [key, { amount: new BigNumber(resp.stx[key]), symbol: 'STX' }])
+  ) as Record<AccountBalanceStxKeys, Money>;
+}
+
+function createStxCryptoAssetBalance(
+  stxMoney: Record<AccountBalanceStxKeys, Money>,
+  inboundBalance: Money,
+  outboundBalance: Money
+): StxCryptoAssetBalance {
+  const totalBalance = createMoney(stxMoney.balance.amount, 'STX');
+  const unlockedBalance = subtractMoney(stxMoney.balance, stxMoney.locked);
+
+  return {
+    availableBalance: subtractMoney(totalBalance, outboundBalance),
+    availableUnlockedBalance: subtractMoney(unlockedBalance, outboundBalance),
+    inboundBalance,
+    lockedBalance: createMoney(stxMoney.locked.amount, 'STX'),
+    outboundBalance,
+    pendingBalance: subtractMoney(sumMoney([totalBalance, inboundBalance]), outboundBalance),
+    totalBalance,
+    unlockedBalance,
+  };
```

Comment:
Here we're performing some business logic, but I kinda don't mind cause it's just simple addition subtraction. Whereas anything more complicated, I'd question where it should live